lms/startup.py

```diff
@@ -198,6 +202,7 @@ def course_end_date_sub(user, course):
         '%%USER_ID%%': Keyword(user_id_sub, 'anonymous_user_id (for use in survey links)'),
         '%%USER_FULLNAME%%': Keyword(user_fullname_sub, 'user profile name'),
         '%%COURSE_DISPLAY_NAME%%': Keyword(course_display_name_sub, 'display name of the course'),
+        '%%COURSE_ID%%': Keyword(get_course_id_string, 'course identifier'),
```

Comment:
here your ```COURSE_ID``` string is different than it was in common/djangoapps/util/keyword_substitution.py